tests_e2e/orchestrator/scripts/get-agent-modules-path

```diff
@@ -18,34 +18,20 @@
 #
 
 #
-# Returns the path to the Python executable.
+# Returns the PYTHONPATH on which the azurelinuxagent and associated modules are located.
+#
+# To do this, the script tries to find the python command used to start the agent and then
+# returns the value of site.getsitepackages().
 #
 set -euo pipefail
 
-# python3 is available on most distros
```

Comment:
Somehow git captures this as a rename + changes. I actually removed find-python and created a totally unrelated script, get-agent-modules-path